epoxy-litho/src/main/java/com/airbnb/epoxy/EpoxyLithoModel.java

```diff
@@ -0,0 +1,88 @@
+package com.airbnb.epoxy;
+
+import android.os.Looper;
+import android.view.ViewGroup;
+
+import com.facebook.litho.Component;
+import com.facebook.litho.ComponentContext;
+import com.facebook.litho.ComponentLifecycle;
+import com.facebook.litho.ComponentTree;
+import com.facebook.litho.LithoView;
+
+import java.util.List;
+
+/**
+ * Experiment support for Facebook's Litho library. Any litho component specs will automatically
+ * have a model generated for them that extends this class.
+ */
+public abstract class EpoxyLithoModel<T extends ComponentLifecycle>
+    extends EpoxyModelWithView<LithoView> {
+
+  private Component<T> component;
+
+  protected abstract Component<T> buildComponent(ComponentContext context);
+
+  public Component<T> getComponent(ComponentContext context) {
+    // TODO: (eli_hart 4/23/17) verify added to controller first?
+    if (component == null) {
+      // TODO: (eli_hart 4/23/17) reuse component context?
+      component = buildComponent(context);
+    }
+    return component;
+  }
+
+  public void clearComponent() {
+    // TODO: (eli_hart 4/23/17) Do this at some point to avoid leaking the context?
+    component = null;
+  }
+
+  @Override
+  public void bind(LithoView view) {
+    Component<T> component = getComponent(view.getComponentContext());
+    if (view.getComponentTree() == null) {
+      view.setComponentTree(ComponentTree.create(view.getComponentContext(), component)
+          .asyncStateUpdates(false) // does this need to be off?
```

Comment:
yep, I noticed that async adapter changes are not yet implemented, but that seems separate from state changes. this seems to only be called by ComponentContext#updateStateAsync which doesn't seem to have usages, so I'm not quite sure how it works, but it seems safer to not do anything async.